### PR TITLE
[RM TSC] Add requirements principles

### PR DESCRIPTION
**Pull request by xmulligan**
recommended to move from #1738 to here by @pgoyal01

### CODE REVIEW

**Comment by pgoyal01** [comment]
@xmulligan Can you move the "Requirements Principles" before the "Architectural Principles"? Thanks

**Comment by xmulligan** [comment]
@pgoyal01 done

File: doc/common/chapter00.md [hunk 1]
```diff
@@ -112,6 +112,19 @@ Any specification work created within CNTT **must** conform to the following pri
    - This pertains to features that operate at detailed levels of hardware granularity, such as EPA.
 
 <a name="2.2"></a>
+## Requirements Principles
+The agreed upon rules and recommendations to which a compliant NF or cloud infrastructure must adhere. 
+
```
Comment (ASawwaf) [inline doc/common/chapter00.md:117]: @xmulligan , Can we replace NF with workload

File: doc/common/chapter00.md [hunk 2]
```diff
@@ -112,6 +112,19 @@ Any specification work created within CNTT **must** conform to the following pri
    - This pertains to features that operate at detailed levels of hardware granularity, such as EPA.
 
 <a name="2.2"></a>
+## Requirements Principles
+The agreed upon rules and recommendations to which a compliant NF or cloud infrastructure must adhere.
```
Comment (xmulligan) [inline doc/common/chapter00.md:116]:
```suggestion
The agreed upon rules and recommendations to which a compliant workload or cloud infrastructure must adhere. 
```

File: doc/common/chapter00.md (hunk 1 above)
Comment (xmulligan) [reply doc/common/chapter00.md:117]: thanks, updated

**Comment by rabi-abdel** [comment]
@ASawwaf re-review please

**Pull request merged**